src/main/kotlin/org/koil/image/AccountImageRepository.kt

```diff
@@ -0,0 +1,8 @@
+package org.koil.image
+
+import org.springframework.data.repository.CrudRepository
+import java.util.*
+
+interface AccountImageRepository : CrudRepository<AccountImage, UUID> {
+    fun findAccountImageByAccountId(accountId: Long): AccountImage?
```

Comment:
Now that I read it in this context, Account Images are probably better known as Profile Pictures.  But that would be a big change and I think we all know what is meant by Account Images.

Reply:
I think it's a worthy change. I'll make it.